src/Take.Blip.Builder/Models/ConditionComparison.cs

```diff
@@ -167,7 +170,7 @@ public static ComparisonType GetComparisonType(this ConditionComparison conditio
                     return (v1, v2) => v1 != null && v2 != null && v1.EndsWith(v2, StringComparison.OrdinalIgnoreCase);
 
                 case ConditionComparison.Matches:
-                    return (v1, v2) => v1 != null && v2 != null && Regex.IsMatch(v1, v2);
+                    return (v1, v2) => v1 != null && v2 != null && Regex.IsMatch(v1, v2, RegexOptions.Compiled | RegexOptions.IgnoreCase, _configuration.RegexTimeout);
```

Comment:
The condition regex is dynamic because the user input it. Compiling this regex will load our CPU

Reply:
Done. Using as Default the RegexOption.None